README.md

```diff
@@ -256,6 +256,26 @@ We have snapshot tests on some of our integration tests. To update those you wil
 UPDATE_SNAPSHOT=true yarn test:integration
 ```
 
+#### Running integration tests in IntelliJ
```

Comment:
Nice, thank you! Don't worry about changing this (not big deal at all) but for info - for commits like this which are useful things we've spotted along the way but not directly related to the ticket, we tend to prefix the commit with "BAU:" rather than the ticket number.